sponsor.html

```diff
@@ -394,11 +511,100 @@ <h2> Sponsorship Prospectus </h2>
     <section class="footer">
       Copyright &copy; 2019 - 2020 Technica. All Rights Reserved.
     </section>
-
   </article>
 </div>
 
 <!-- mlh badge -->
 <a id="mlh-trust-badge" style="display:block;max-width:100px;min-width:60px;position:fixed;right:50px;top:0;width:10%;z-index:10000" href="https://mlh.io/seasons/na-2020/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2020-season&utm_content=white" target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/2020/mlh-trust-badge-2020-white.svg" alt="Major League Hacking 2020 Hackathon Season" style="width:100%"></a>
+
+<script>
+/* initializing tooltips */
+$(function () {
+  $('[data-toggle="tooltip"]').tooltip()
+})
+
+/* [modified] plugin to have text in the middle of donut from: stackoverflow.com/questions/20966817 */
+Chart.pluginService.register({
```

Comment:
Can we move this to a separate JS file for improved readability?

Reply:
yeah that makes sense